### ISSUE TITLE
meme competition

### ISSUE THREAD

**Issue by steven-hh-ding** [CISC-873/Information#2]
Please share any Data Mining/Data Science/Machine Learning memes here!! (the one got the most emoji will be used on the front page, approved by the instructor)

**Comment by 16arqueensu** [comment]
![image](https://user-images.githubusercontent.com/68115584/92496007-94788880-f1c5-11ea-8d50-23bb01142afd.png)

**Comment by tis13** [comment]
![mlawiab](https://user-images.githubusercontent.com/47393558/92499205-2050db80-f1eb-11ea-9214-d1137c01a8de.jpg)

**Comment by tis13** [comment]
![20200908_131123798_iOS](https://user-images.githubusercontent.com/47393558/92499255-2f378e00-f1eb-11ea-81c8-b579662fa2a9.jpg)

**Comment by steven-hh-ding** [comment]
current winner: https://github.com/CISC-873/Information/issues/2#issuecomment-688951476
[refreshed every 24 hours]

**Comment by ChrisJMolloy** [comment]
![meme](https://user-images.githubusercontent.com/25777235/92512648-17590d80-f1dd-11ea-82f6-36dec65aa489.png)

**Comment by amirsojoodi** [comment]
![photo_2020-09-08_22-10-27](https://user-images.githubusercontent.com/10928452/92546260-21503000-f220-11ea-8792-5a804fe72160.jpg)

**Comment by steven-hh-ding** [comment]
lol i love this

**Comment by ansariminhaj** [comment]
![DrVNuF8WkAAGLkN](https://user-images.githubusercontent.com/46193078/92551638-7d20b600-f22c-11ea-9dbd-0dc495e64f0d.jpg)

**Comment by MA171** [comment]
![image](https://user-images.githubusercontent.com/20342539/92641512-8b7fd800-f2e7-11ea-840d-46306bdafe1b.png)

**Comment by steven-hh-ding** [comment]
current winner: https://github.com/CISC-873/Information/issues/2#issuecomment-688970192 refreshed.

**Comment by tylermainguy** [comment]
![image](https://user-images.githubusercontent.com/29045168/92747167-ef3eeb00-f351-11ea-9105-3122d15c48e7.png)

**Comment by jackliuto** [comment]
![ypjdt1anuej41](https://user-images.githubusercontent.com/14166685/92927491-b1060200-f40b-11ea-82dd-3b44e62924a0.jpg)

**Comment by moatary** [comment]
Qualified MLers are rare though
![1](https://user-images.githubusercontent.com/51672542/92947393-2eae2a00-f46d-11ea-8df0-3ed2e0aba5a3.png)

**Comment by KathleenAb** [comment]
![is this a machine learn](https://user-images.githubusercontent.com/22873585/92953175-4797ea80-f42f-11ea-8ccf-8cc3d01a4ea5.jpg)
Clickbait AI articles be like:

**Comment by steven-hh-ding** [comment]
lol exactly one of the topic in our lecture. the last one. i should put it there

**Comment by AndreaPerera** [comment]
idk if this counts but it's my favourite 

![IMG_0250](https://user-images.githubusercontent.com/23272253/92970022-fd713200-f44b-11ea-98be-494d63a0804d.JPG)

**Comment by amirsojoodi** [comment]
![photo_2020-09-11_20-19-53](https://user-images.githubusercontent.com/10928452/92982489-59e44980-f46c-11ea-8ee9-cdd153591e3c.jpg)

**Comment by amirsojoodi** [comment]
![photo_2020-09-11_20-25-55 - Copy](https://user-images.githubusercontent.com/10928452/92982760-b98f2480-f46d-11ea-83d1-a64e217431a4.jpg)

**Comment by steven-hh-ding** [comment]
Lol this make my day thank you!

**Comment by tylermainguy** [comment]
![image](https://user-images.githubusercontent.com/29045168/93383843-35a0b800-f832-11ea-9719-1622b19c3a7f.png)

**Comment by b-safwat** [comment]
![1_qMUdkZf5cPFy5MHyybqTOQ](https://user-images.githubusercontent.com/33165067/93594512-44b07300-f9b6-11ea-8561-65257018dfa8.png)

**Comment by moatary** [comment]
I think  Einstein can teach Locally linear Embedding to his Grandmo. But can Einstein teach her Einstein's Manifold?

![udare](https://user-images.githubusercontent.com/51672542/93622177-eacd9e80-f9f1-11ea-831b-bf5312a0b529.jpg)

**Comment by steven-hh-ding** [comment]
@moatary did you ask the last question Mojtaba? sorry didn't get enough time to discuss with you though
```
Sorry if irrelevant to the context. But, why ERM/SRM are not Ocaam Razor friendly? 
When we are dealing with overfitting, why we couldnt just avoid sacrificing ocaam razor?
```
The issue is that for a lot of problems these days, complex but sparse (regulated) solutions can be trained much easier. But of course if not carefully they will overfit instead. There are much compressed/simple solution that can achieve the same but difficult to learn, but difficult to come up at the first place. I still prefer starting with simplest solutions though. Not sure if I understand your question correctly though

**Comment by moatary** [comment]
Yes it was me. Sorry I should not be that impatient to ask it then, as you were tired and question was not relevant. 
Exactly I got my answer. As you mentioned, current GD framework won't let us learn from simple to complex. So maybe it is better to constrain regularizations like sparsity after a while, when model progresses, and not right from the beginning of training. What fascinates me when I compare brain to current AI, is the power of the Brain to allocate neurons in proportion to the salient nonlinearity of input data in general. So that's why I got curious why current ML models do exactly reverse of what the Brain does. Brain's most important bias is least effort law (Occaam's Razor), while current ML focuses more on No Free Lunch, learning all neurons at once in GD, buying lots of issues like local minima, gradient vanishing/exploding and etc...

**Comment by b-safwat** [comment]
![119519154_3291534004228788_296137329967678064_n](https://user-images.githubusercontent.com/33165067/93666164-29a83680-fa7c-11ea-895f-42f2ecb1ac00.jpg)

**Comment by moatary** [comment]
One hot or one-hot
![11](https://user-images.githubusercontent.com/51672542/93738219-17e89f80-fbf2-11ea-8947-85c0df561873.jpg)

**Comment by JifanYang** [comment]
![v2-f666a6b1522fe2b0865634ad2f9dc1fb_720w](https://user-images.githubusercontent.com/70959382/94257920-a183ce80-ff5e-11ea-91a3-12cf1029fd25.jpg)

**Comment by Gitsamshi** [comment]
![1_x7P7gqjo8k2_bj2rTQWAfg](https://user-images.githubusercontent.com/19702190/94319206-24555b00-ff58-11ea-8a5b-4b95db000e17.jpeg)

**Comment by moatary** [comment]
I wish we had second meme competition in which we embed and pose Data Mining and AI challenges in terms of open questions.
That would be not only awesome, but also instructive. The "instructive meme" types cannot be as funny as memes in this page, but they create challenges and motivations to learn. 

My suggestion for another page:
- Participants state their AI related questions in terms of memes, artworks, poems and funny analogy.
- Other participants answer to the questions.
- The answer with the most emoji (and also questions with most emoji) are selected as the winner

Example: My following post:

**Comment by moatary** [comment]
Is finding the pattern of cat in the dots representing "creativity" or indeed, enforcing overfitted structure to the dots?

"Q FOR COMPETITION!!": If it is not creativity, then what is the best definition for "creativity"?

![weewee](https://user-images.githubusercontent.com/51672542/94332077-c3249a80-ffde-11ea-929a-d0414aba35f0.png)

**Comment by henrywho16** [comment]
Truth of Datamining
https://www.theraleighregister.com/truthofdatamining.html

**Comment by ChrisJMolloy** [comment]
Assignment 0 🥺

![20201001_203130](https://user-images.githubusercontent.com/25777235/94876284-37b76900-0425-11eb-81ca-063300794f64.jpg)

**Comment by moatary** [comment]
The eigen-faces of Monroe generated by an artist. Are they statistically independent?

![25](https://user-images.githubusercontent.com/51672542/94931387-91c33780-04d4-11eb-9a98-2c3adf98dbe4.jpg)

**Comment by MA171** [comment]
Assignment 0 new submissions be like :
![image](https://user-images.githubusercontent.com/20342539/95266729-7477cb80-083c-11eb-890d-d433311c5990.png)

**Comment by moatary** [comment]
Features, models, and hyper-parameters saved my life
How do u feel/fill in the blanks?!

![progress1](https://user-images.githubusercontent.com/51672542/95670886-dff3cd00-0b9d-11eb-89c1-a98c6bae8a5e.png)

**Comment by llauriez** [comment]
I wanted to take this class, but I couldn't. It is fun to join the meme competition tho. Saw this on reddit,

![122977465_10161075098991840_8206673527520466908_n](https://user-images.githubusercontent.com/40523200/97792557-2c876280-1bb6-11eb-87be-f290a91d6555.jpg)

**Comment by KathleenAb** [comment]
![image](https://user-images.githubusercontent.com/22873585/99158932-0b3e7000-26a6-11eb-82ad-0827722c04bf.png)

**Comment by AndreaPerera** [comment]
hi school is hard so i made this tag yourself meme instead 
<img width="915" alt="Screenshot 2020-11-15 at 12 11 05 AM" src="https://user-images.githubusercontent.com/23272253/99171499-8d935800-26d7-11eb-9d0c-bee5913689c2.png">

**Comment by moatary** [comment]
![Capture+_2020-11-22-09-38-29](https://user-images.githubusercontent.com/51672542/99896610-645e6300-2ca7-11eb-9da5-3648186c9799.png)

**Comment by ryanz8** [comment]
![image](https://user-images.githubusercontent.com/32717024/100187628-7b26d480-2eb6-11eb-99ec-9f20182bfd4a.png)
